greengrass_defender_agent/agent.py

```diff
@@ -45,6 +47,45 @@ def set_configuration(configuration):
     return new_config
 
 
+def set_env_variables_config():
+    env_config = {}
+    if environ.get(config.CONNECT_AND_PUBLISH_RETRY_CONFIG_KEY):
```

Comment:
Isn't this `PUBLISH_RETRY_CONFIG_KEY` in config.py?

Reply:
Yeah. Thanks for catching that. I make sure the config key name matches in the new revision and deployed the version to my testing EC2. Cut off the internet to force it retry and observe that if can pick up the override retry count values from system env.